packages/mobile/__mocks__/src/geth/network-config.ts

```diff
@@ -0,0 +1,7 @@
+export default {
```

Comment:
Funny, I actually added this same thing here: https://github.com/celo-org/celo-monorepo/pull/1209